kr8s/tests/test_auth.py

```diff
@@ -177,6 +177,21 @@ def test_no_config():
         kr8s.api(kubeconfig="/no/file/here")
 
 
+def test_kubeconfig_multi_paths(k8s_cluster, tmp_path):
+    kubeconfig1: Path = k8s_cluster.kubeconfig_path
+    kubeconfig2 = Path(tmp_path / "kubeconfig").write_bytes(kubeconfig1.read_bytes())
+    kubeconfig_multi_str = f"{kubeconfig1}:{kubeconfig2}"
+    api = kr8s.api(kubeconfig=kubeconfig_multi_str)
+    assert (
+        str(api.auth._serviceaccount) == "/var/run/secrets/kubernetes.io/serviceaccount"
+    )
```

Comment:
This seems almost identical to the existing `test_kubeconfig_multi` test on line 103. This new test is better because it duplicates the config, so I would be in favour of replacing the existing test with this one.

Reply:
Oh I missed `test_kubeconfig_multi`. I think they are different cases - "same path found twice" vs "two diff paths" - so may as well keep both? More testing always better than less imo.

Reply:
I think the problem is the existing test isn't intentionally testing the same path twice. It actually loads both which I'm not sure it should be doing. I just used the same path twice out of laziness. I think the intent of both tests is the same thing, so we should remove the existing one.

If you have a desire to add a new test that tests the same path twice (and checks that it is correctly deduplicated) then that would be great asa follow up!